.github/workflows/reviewdog-workflow.yml

```diff
@@ -276,6 +276,11 @@ jobs:
           path: ${{ steps.py-changes.outputs.py_files_list }}
         env:
           GITHUB_TOKEN: ${{ secrets.github_token }}
+      - name: Info for local execution
+        if: failure()
+        run: |
+          echo "Running linting tools failed. This should be equivalent to running './lte/gateway/python/precommit.py --lint --diff' locally."
```

Comment:
Is this actually equivalent given that the wemake-python-styleguide action outputs comments for entire edited files? Is this what `./lte/gateway/python/precommit.py --lint --diff` does?

Reply:
My understanding is that the wemake styleguide was added to the docker container running with the `precommit.py` with #14994 and that the edited files are checked entirely. Or am I mistaken @LKreutzer?

Reply:
Locally only the last commit is checked and in CI everything up to the merge-base, but otherwise they should be identical.

Reply:
I just tested this locally with two commits in which I introduced linting errors, and the errors for both commits were reported, so it looks good to me.